src/broker/broker_proxy_conn.c

```diff
@@ -371,6 +376,14 @@ broker_find_available_proxy (T_SHM_PROXY * shm_proxy_p)
 	  min_cur_client = cur_client;
 	}
     }
+
+  if (shm_proxy_p->num_proxy > 0 && fd < 0 && retry_count++ < PROXY_SVR_CON_RETRY_COUNT)
+    {
+      pthread_mutex_unlock (&proxy_conn_mutex);
```

Comment:
pthread_mutex_lock() only be used at not window.
So, you should use #if !defined(WINDOWS) .. #endif.

Reply:
You right!
Thank you for your comment!